src/utils/d2.ts

```diff
@@ -19,6 +25,11 @@ export const d2BaseModelDetails = [
     { name: "href", text: i18n.t("API link") },
 ];
 
+export const organisationUnitsDetails = [
+    { name: "level", text: i18n.t("Level"), sortable: true },
+    ...d2BaseModelDetails,
```

Comment:
Not sure, do we mind that *Level* is the first item instead of the traditional *Name*?

Reply:
Maintenance doesn't even add the level to details, just to columns.

@adrianq Would you remove it from details too?

Reply:
As discussed with Adrian, we will keep it and lower it down.